modules/graphdb/variables.tf

```diff
@@ -195,6 +195,12 @@ variable "graphdb_image_id" {
 
 # GraphDB VM
 
+variable "node_count_key_name" {
```

Comment:
The rest of the variables follow `graphdb_<property>_name` when referring to app config.

Reply:
updated